lib/matplotlib/patches.py

```diff
@@ -511,10 +511,6 @@ def _bind_draw_path_function(self, renderer):
         gc.set_url(self._url)
         gc.set_snap(self.get_snap())
 
-        rgbFace = self._facecolor
```

Comment:
where did the logic this used to support get moved too?  This removal makes me nervous just because it may be hilighting issues else where...

Reply:
Originally, this had two usages:
~~~
        if self.get_path_effects():
            for path_effect in self.get_path_effects():
                path_effect.draw_path(renderer, gc, tpath, affine, rgbFace)
        else:
            renderer.draw_path(gc, tpath, affine, rgbFace)
~~~

The first has been removed [here](https://github.com/timhoffm/matplotlib/commit/09d5f5ff6ead092ea0debc6e4d746e640f41329f#diff-d6593efaf6f4a6b9fb34f84bfa9964b5) and the second [here](https://github.com/timhoffm/matplotlib/commit/5c564d4007aaa02aaf671df1503e5e0aca61ed47#diff-d6593efaf6f4a6b9fb34f84bfa9964b5)

Reply:
:+1: Thanks for doing the forensics on that !

Reply:
For my PR's side I don't think there's any problem.